index.css

```diff
@@ -0,0 +1,1147 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", Helvetica, sans-serif;
+}
+
+.navbarimg {
+  margin-left: 20px;
+  margin-top: 15px;
+  z-index: 100;
+  position: absolute;
+}
```

Comment:
- [ ] [OPTIONAL] You have done a great job ensuring the design and implementation of the hamburger menu 👍 👏 . However, we should stick to the original design as much as possible ☺️ . Therefore I suggest that you change the color of the hamburger to match the original design.

Original design 😎 
![image](https://user-images.githubusercontent.com/99394374/219880612-7b951f8a-f1c3-4e30-8ae6-bb29b91f23e3.png)

Your design 😎 
![HUMBER](https://user-images.githubusercontent.com/99394374/219880651-353f6304-9747-4f70-8048-8e985c67a4e4.png)